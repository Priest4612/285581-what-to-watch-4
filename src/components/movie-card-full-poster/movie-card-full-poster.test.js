import React from 'react';
import renderer from 'react-test-renderer';

import filmsMock from '../../mocks/test-films-mock.json';

import MovieCardFullPoster from './movie-card-full-poster.jsx';


it(`Render MovieCardFullPoster`, () => {
  const {name, posterImage} = filmsMock[2];
  const tree = renderer
    .create(<MovieCardFullPoster
      name={name}
      posterImage={posterImage}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
