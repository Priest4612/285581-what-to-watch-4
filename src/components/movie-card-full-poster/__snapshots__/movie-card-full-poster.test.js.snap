// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCardFullPoster 1`] = `
<div
  className="movie-card__poster movie-card__poster--big"
>
  <img
    alt="Bohemian Rhapsody"
    height="327"
    src="img/bohemian-rhapsody.jpg"
    width="218"
  />
</div>
`;
